cs/mailing.texy

```diff
@@ -179,7 +192,24 @@ $mailer->send($mail);
 \--
 
 
-Můžeme si také vytvořit vlastní mailer, jde o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+Fallback Mailer
+---------------
+
+Máme-li občas potíže s odesíláním emailů (např. výpadek smtp serveru), můžeme využít `FallbackMailer`, který umí v případě chyby automaticky zopakovat pokus o odeslání, buď pomocí stejného nebo pomocí alternativního maileru.
```

Comment:
Trošku bych to upravil, aby bylo jasné, že FallbackMailer nic přímo neodesílá.

Třeba: můžeme využít `FallbackMailer`, který sám přímo emaily neodesílá, ale odeslání zprostředkovává přes sadu mailerů. Pokud jeden selže, zopakuje pokus u dalšího, a když další není, tak u stejného maileru. Kromě sady mailerů lze nastavit nastavit počet pokusů pro odeslání a čekací dobu mezi pokusy.